KIPs/kip-34.md

```diff
@@ -307,7 +307,9 @@ Each keyring class uses the `PrivateKey` class, which has one private key as a m
 
 `Keystore` is a class that contains encrypted keyring. The internally defined value differs depending on the keystore version, see [KIP-3](./kip-3.md).
 
-`KeyringContainer` is an "in-memory wallet" class that manages keyring instances. It manages the keyring instances based on their addresses.
+In the wallet layer, an interface called `IWallet` is defined, and this interface is an interface of `Wallet` that stores data used when signing a transaction.
```

Comment:
```suggestion
An interface `IWallet` contains functions related to signing a transaction.
```

Is there the class `Wallet` somewhere?

Reply:
I used the term Wallet as the concept of a class that stores data used when signing. In caver, Wallet corresponds to KeyringContainer. 

I described here with an `account`, which is a data unit stored here, and a `wallet` that manages and stores it, but if there is a more clear description method, please tell me. ;-;

Reply:
If wallet is not an actual class, it would be better to use wallet instead of `wallet`.
I think my suggestion is enough to explain `IWallet`. What do you think?

Reply:
Oh, I understood what you were saying. It looked like a real class by processing a block of code. Including the suggestions given above, I uploaded a separate [commit](https://github.com/klaytn/kips/pull/53/commits/f3152a29e248009a39314722a33b1e960d400f12) and modified it.